android/java/org/chromium/chrome/browser/app/helpers/ImageLoader.java

```diff
@@ -72,19 +77,23 @@ public class ImageLoader {
     /**
      * Downloads an image from a given URL, including support for GIF and SVG image types.
      * @param url URL of the image to download.
-     * @param context Android context used by Glide for applying transformations.
+     * @param RequestManager Glide request manager for applying transformations.
      * @param isCircular When {@code true}, a circular transformation will be applied.
+     * @param roundedCorners Radius of the circle used to round the corners in dip. Unused when
+     *         {@code isCircular} is {@code true}.
      * @param imageView ImageView where the downloaded image will be set.
      * @param callback Callback used to notify if the image has been set correctly. It can be {@code
      *         null}.
      */
-    public static void downloadImage(String url, final Context context, final boolean isCircular,
-            final ImageView imageView, final Callback callback) {
+    public static void downloadImage(String url, final RequestManager requestManager,
+            final boolean isCircular, final int roundedCorners, final ImageView imageView,
+            final Callback callback) {
         if (!isValidImgUrl(url)) {
             if (callback != null) callback.onLoadFailed();
             return;
         }
 
+        Resources resources = ContextUtils.getApplicationContext().getResources();
```

Comment:
Not necessarily, unless you see any side effects I'm not seeing right now. 
Because we need access to the resources we can use application context (not activity context), so we are safe here.